glusterd2/commands/global/getglobaloptions.go

```diff
@@ -0,0 +1,46 @@
+package globalcommands
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/gluster/glusterd2/glusterd2/cluster"
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/glusterd2/store"
+	"github.com/gluster/glusterd2/pkg/api"
+	"github.com/gluster/glusterd2/pkg/errors"
+	"github.com/gorilla/mux"
+)
+
+func getGlobalOptionsHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Read default options to temp opts, then get from store and update temp opts and return. Good to return the following details.

     Name
     Value
     Default Value
     ModifiedFlag